windows/tools/OobSimulator/App.xaml.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.

```

Comment:
Could you run the auto-formatter on this and other files?

Reply:
I don't have the clang-tidy nor clang-format files in this project, but I can add them

Reply:
.clang-tidy and .clang-format are both present in the top-level of nearobject-framework, so I believe it is possible to right-click this file in VSCode and run the auto-formatter.

Reply:
Oh I see. I ended up adding them in and running the formatter, but I had to add "clang-format off"/"clang-format on" in a couple of places due to the precompiled headers that are automatically part of this project

Reply:
But it looks like it didn't catch everything for some reason, so I need to go back and fix some things